### ISSUE TITLE
Recording module failed with error

### ISSUE THREAD

**Issue by bennals** [JayVora-SerpentCS/SerpentCS_Contributions-v8#174]
I tried to add a simple char field to res_partner (allergies) and received the following error.

Traceback (most recent call last):
  File "/home/odoo/odoo-server/odoo/http.py", line 646, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo/odoo-server/odoo/http.py", line 307, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/home/odoo/odoo-server/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/home/odoo/odoo-server/odoo/http.py", line 683, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo/odoo-server/odoo/http.py", line 339, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo/odoo-server/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo/odoo-server/odoo/http.py", line 332, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/odoo/odoo-server/odoo/http.py", line 927, in __call__
    return self.method(*args, **kw)
  File "/home/odoo/odoo-server/odoo/http.py", line 512, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/odoo-server/addons/web/controllers/main.py", line 934, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo/odoo-server/addons/web/controllers/main.py", line 922, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/odoo/odoo-server/odoo/api.py", line 687, in call_kw
    return call_kw_model(method, model, args, kwargs)
  File "/home/odoo/odoo-server/odoo/api.py", line 672, in call_kw_model
    result = method(recs, *args, **kwargs)
  File "/home/odoo/thirdparty/addons/base_module_record/wizard/base_module_record_objects.py", line 133, in inter_call
    res = base_module_save._create_module(self, data)
  File "/home/odoo/thirdparty/addons/base_module_record/wizard/base_module_save.py", line 33, in _create_module
    s = StringIO.StringIO()
AttributeError: type object '_io.StringIO' has no attribute 'StringIO'

This is potentially an enormously useful module and I'm not sure why it hasn't generated more interest as an OCA alternative to Odoo Studio for CE, particularly as it has existed for such a long time.  I can't help feeling though that some improved instructions, or even better, a demonstration video, would make a big difference to the situation.

**Comment by JayVora-SerpentCS** [comment]
Pull with latest source please. thanks.

**Comment by bennals** [comment]
@JayVora-SerpentCS 
Thanks.  I pulled the latest source and now that error does not occur, but there still appears to be a problem.

When I click 'Record' the 'Thanks For using Module Recorder' message pops up but I can't see that anything else has occurred.  There is no .zip file created that I can find and I tailed the log at /var/log/odoo-server/odoo.log and couldn't see anything that would indicate where the .zip file might be.

I suspect I'm not using it correctly but some instructions would help;)
